src/css/base.css

```diff
@@ -17,6 +17,7 @@
 
   --tablet-container-max-width: 704px;
   --desktop-container-max-width: 1088px;

```

Comment:
As padding is 64px, should be 1216

Reply:
@AntonChubarov this is only for 2 secctions, all other should be 1088.
So it's better to leave as is the default. And for 2 srctions (that are wider) need to override that value to 1216

Reply:
Ok, than need to be overrided